test/check-machines-migrate

```diff
@@ -81,7 +81,7 @@ class TestMachinesMigration(VirtualMachinesCase):
 
         # Set Static Ports for NFS v3 on Ubuntu 20.x
         if machine1.image in ["ubuntu-2004"]:
-            machine1.execute("sed -i 's/--manage-gids/--manage-gids -p 20048/' /etc/default/nfs-kernel-server ")
+            machine1.execute("sed -i 's/--manage-gids/--manage-gids -p 20048/' /etc/default/nfs-kernel-server && systemctl restart nfs-kernel-server")
```

Comment:
What does this do?

Reply:
Makes sure that the port rpc.mountd is using is static. It's static on fedora but on ubuntu it's not. So when we use firewall-cmd to open a service, firwall-cmd --add-service mountd this is acting only on specific port, in this case 20048.

Reply:
It's needed for the shared storage tests on ubuntu when firewall is running.